subprojects/gradle/cd/src/test/kotlin/com/avito/ci/CiStepsDynamicTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.avito.ci
+
+import com.avito.test.gradle.file
+import com.avito.test.gradle.gradlew
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+internal class CiStepsDynamicTest {
+
+    @Test
+    fun `cd plugin - custom task created`(@TempDir projectDir: File) {
+        projectDir.file(
+            name = "build.gradle",
+            content = """
+            plugins {
+                id("com.avito.android.cd")
+            }
+            
+            builds {
+                myCustomTask {
+                    description.set("My customTask description")
+                }
+            }
+        """.trimIndent()
+        )
+
+        gradlew(projectDir, "tasks", "-Pci=true").assertThat()
```

Comment:
Could you add a negative scenario?

Reply:
Which one exactly? Calling non existing  task? I don't see much value in it

Reply:
That with `ci=false` we won't have any task.